docs/admin/index.md

```diff
@@ -38,7 +38,7 @@ the Unix socket `unix:///var/run/docker.sock`
 
 ### Configuring the docker daemon directly
 
-If you're running the `docker` daemon directly by running `docker daemon` instead
+If you're running the `docker` daemon directly by running `dockerd` instead
```

Comment:
Would be better as "running the Docker daemon" here, as `docker` is confusing now `docker` is not a daemon. (also two lines down)